src/pages/projects/[slug]/index.tsx

```diff
@@ -72,10 +80,10 @@ export const getStaticProps = async ({ params, preview }) => {
       content: await addAdditionalInfoToBlocks(project.content),
       opengraph: project.opengraph ?? {},
       relatedProjects: await Promise.all(
-        project.relatedProjects?.map(async related => ({
+        project.relatedProjects?.map(async (related) => ({
```

Comment:
Did we change the linting rules?

Reply:
Nope, just the default formatter wasn't specificed for typescript, so my editor used Prettier instead of the typescript formatter.
Have reverted this, and added the default formatter in editor settings